cpp/src/io/csv/csv_common.h

```diff
@@ -50,6 +50,7 @@ struct stats {
   uint32_t countInt32;
   uint32_t countInt64;
   uint32_t countNULL;
+  uint32_t negative_integer_instance;
```

Comment:
We do have style guidelines that say use "snake case" and not CamelCase...

Reply:
I know, and should have explained the suggestion.
Recommended this since we'll be cleaning this up anyway. Consistency helps with the clean up.
I would like the most for the other members to be renamed to snake_case, but it's a outside of this PR.